src/server/system_services/pool_server.js

```diff
@@ -1131,37 +1126,39 @@ async function get_optimal_non_mongo_pool_id() {
 
 async function update_account_default_resource() {
     for (;;) {
-        try {
-            const optimal_pool_id = await get_optimal_non_mongo_pool_id();
-
-            if (optimal_pool_id) {
-                const updates = system_store.data.accounts
-                    .filter(account =>
-                        account.email.unwrap() !== config.OPERATOR_ACCOUNT_EMAIL &&
-                        account.default_pool &&
-                        _is_mongo_pool(account.default_pool)
-                    )
-                    .map(account => ({
-                        _id: account._id,
-                        $set: {
-                            'default_pool': optimal_pool_id
-                        }
-                    }));
+        const system = system_store.data.systems[0];
+        if (system) {
```

Comment:
I would move this code inside the try block.

Reply:
no need as we know system_store is already initiated - but in order to ease up you mind